packages/terra-heading/src/terra-dev-site/doc/example/HeadingLevels.jsx

```diff
@@ -4,6 +4,13 @@ import Heading from 'terra-heading';
 import classNames from 'classnames/bind';
 import styles from './HeadingDocCommon.module.scss';
 
+// Terra includes styling for all standard headings `h1` through `h6`, as well as styles that match the size of their respective heading. Headings should be
```

Comment:
I think these comments would be better placed as a description for the examples in the mdx file.

Reply:
Waiting for [this fix](https://github.com/cerner/terra-dev-site/blob/master/CHANGELOG.md#fixed) to be released.